core/src/test/resources/invalid_json.json

```diff
@@ -0,0 +1 @@
+AAAAAAAAAAA
```

Comment:
under `src/test/resources` there is already a folder called `swagger` to contain swagger schemas, with several sub-folders. move those 3 files there, where appropriate, eg can create new folder `src/test/resources/swagger/urlissue`